security-insights-schema-1.0.0.yaml

```diff
@@ -130,6 +130,35 @@ properties:
                 $id: '#/properties/contribution-policy/properties/accepts-bot-pull-requests'
                 description: 'Define if the maintainers accept pull-requests generated by bots or automated tools.'
                 type: boolean
```

Comment:
Can we indicate somewhere that this is `true` by default unless explicitly communicated?

Reply:
I would also not make `accepts-bot-pull-requests` a required field

Reply:
I would also not make `accepts-pull-requests` a required field either

Reply:
> I would also not make `accepts-bot-pull-requests` a required field
> I would also not make `accepts-pull-requests` a required field either

Why? To remove frictions to the maintainers when they create this spec? I would suggest keeping them required because a boolean value is easy to set but in this way, if you set it on `true`, and you don't have a `CONTRIBUTING.md`, contributors have an additional reason for asking maintainers to write it.

> Can we indicate somewhere that this is `true` by default unless explicitly communicated?

I was thinking about this, and I am not sure if this should be a scanner choice or not. Setting these keys as required helped me to not make this decision.

Reply:
> Why? To remove frictions to the maintainers when they create this spec?

Yes, but also because this is, inherently, the default for most repositories. Why have an OSS repository if you aren't looking for external contributors?

>  if you set it on true, and you don't have a CONTRIBUTING.md, contributors have an additional reason for asking maintainers to write it.

Why? What encouragement exists?